cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/actions/organizations/CreateOrganization.java

```diff
@@ -0,0 +1,180 @@
+                                       @Param(value = SESSION_TIMEOUT, description = SESSION_TIMEOUT_DESC) String sessionTimeout,
+                                       @Param(value = SESSION_REMEMBER, description = SESSION_REMEMBER_DESC) String sessionRemember,
+                                       @Param(value = COLLABORATOR_AUTH_POLICY, description = COLLABORATOR_AUTH_POLICY_DESC) String collaboratorAuthPolicy,
+                                       @Param(value = COST_ESTIMATION_ENABLED, description = COST_ESTIMATION_ENABLED_DESC) String costEstimationEnabled,
+                                       @Param(value = OWNERS_TEAM_SAML_ID,  description = OWNERS_TEAM_SAML_ID_DESC) String ownersTeamSamlRoleId,
+                                       @Param(value = REQUEST_BODY, description = ORGANIZATION_REQUEST_BODY_DESC) String requestBody,
+                                       @Param(value = PROXY_HOST, description = PROXY_HOST_DESC) String proxyHost,
+                                       @Param(value = PROXY_PORT, description = PROXY_PORT_DESC) String proxyPort,
+                                       @Param(value = PROXY_USERNAME, description = PROXY_USERNAME_DESC) String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true, description = PROXY_PASSWORD_DESC) String proxyPassword,
+                                       @Param(value = TRUST_ALL_ROOTS, description = TRUST_ALL_ROOTS_DESC) String trustAllRoots,
+                                       @Param(value = X509_HOSTNAME_VERIFIER, description = X509_DESC) String x509HostnameVerifier,
+                                       @Param(value = TRUST_KEYSTORE, description = TRUST_KEYSTORE_DESC) String trustKeystore,
+                                       @Param(value = TRUST_PASSWORD, encrypted = true, description = TRUST_PASSWORD_DESC) String trustPassword,
+                                       @Param(value = CONNECT_TIMEOUT, description = CONNECT_TIMEOUT_DESC) String connectTimeout,
+                                       @Param(value = SOCKET_TIMEOUT, description = SOCKET_TIMEOUT_DESC) String socketTimeout,
+                                       @Param(value = KEEP_ALIVE, description = KEEP_ALIVE_DESC) String keepAlive,
+                                       @Param(value = CONNECTIONS_MAX_PER_ROUTE, description = CONN_MAX_ROUTE_DESC) String connectionsMaxPerRoute,
+                                       @Param(value = CONNECTIONS_MAX_TOTAL, description = CONN_MAX_TOTAL_DESC) String connectionsMaxTotal,
+                                       @Param(value = RESPONSE_CHARACTER_SET, description = RESPONSE_CHARACTER_SET_DESC) String responseCharacterSet) {
+
+        organizationDescription = defaultIfEmpty(organizationDescription, EMPTY);
+        requestBody = defaultIfEmpty(requestBody, EMPTY);
+        sessionTimeout = defaultIfEmpty(sessionTimeout,"20160");
```

Comment:
Define **sessionTimeout** as Constant variable and use it. (Like DEFAULT_PROXY_PORT)
And same with **collaboratorAuthPolicy**

Reply:
Changes done.